packages/cli/src/workflow-runner.ts

```diff
@@ -203,6 +211,7 @@ export class WorkflowRunner {
 	}
 
 	/** Run the workflow in current process */
+	// eslint-disable-next-line complexity
```

Comment:
This is necessary for now. Once we remove the old partial execution logic this function will be much simpler.